pillarbox-demo-shared/src/main/java/ch/srgssr/pillarbox/demo/shared/data/Playlist.kt

```diff
@@ -38,6 +41,370 @@ data class Playlist(val title: String, val items: List<DemoItem>, val descriptio
+                    description = "16x9 aspect ratio, H.264 and HEVC @ 30Hz and 60Hz"
+                ),
+                DemoItem(
+                    title = "Apple WWDC Keynote 2023",
+                    uri = "https://events-delivery.apple.com/0105cftwpxxsfrpdwklppzjhjocakrsk/m3u8/vod_index-PQsoJoECcKHTYzphNkXohHsQWACugmET.m3u8"
+                ),
+                DemoItem(
+                    title = "Apple Dolby Atmos",
+                    uri = "https://devstreaming-cdn.apple.com/videos/streaming/examples/adv_dv_atmos/main.m3u8"
+                ),
+                DemoItem(
+                    title = "The Morning Show - My Way: Season 1",
+                    uri = "https://play-edge.itunes.apple.com/WebObjects/MZPlayLocal.woa/hls/subscription/playlist.m3u8?cc=CH&svcId=tvs.vds.4021&a=1522121579&isExternal=true&brandId=tvs.sbd.4000&id=518077009&l=en-GB&aec=UHD"
+                ),
+                DemoItem(
+                    title = "The Morning Show - Change: Season 2",
+                    uri = "https://play-edge.itunes.apple.com/WebObjects/MZPlayLocal.woa/hls/subscription/playlist.m3u8?cc=CH&svcId=tvs.vds.4021&a=1568297173&isExternal=true&brandId=tvs.sbd.4000&id=518034010&l=en-GB&aec=UHD"
+                )
+            )
+        )
+        private val thirdPartyStreams = Playlist(
+            title = "Third-party streams",
+            items = listOf(
+                DemoItem(
+                    title = "Brain Farm Skate Phantom Flex",
+                    uri = "https://sample.vodobox.net/skate_phantom_flex_4k/skate_phantom_flex_4k.m3u8",
+                    description = "4K video"
+                )
+            )
+        )
+        private val bitmovinStreams = Playlist(
+            title = "Bitmovin streams streams",
+            items = listOf(
+                DemoItem(
+                    title = "Multiple subtitles and audio tracks",
+                    uri = "https://bitmovin-a.akamaihd.net/content/sintel/hls/playlist.m3u8"
+                ),
+                DemoItem(
+                    title = "4K, HEVC",
+                    uri = "https://cdn.bitmovin.com/content/encoding_test_dash_hls/4k/hls/4k_profile/master.m3u8"
+                ),
+                DemoItem(
+                    title = "VoD, single audio track",
+                    uri = "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8"
+                ),
+                DemoItem(
+                    title = "AES-128",
+                    uri = "https://bitmovin-a.akamaihd.net/content/art-of-motion_drm/m3u8s/11331.m3u8"
+                ),
+                DemoItem(
+                    title = "AVC Progressive",
+                    uri = "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/MI201109210084_mpeg-4_hd_high_1080p25_10mbits.mp4"
+                )
+            )
+        )
+        private val unifiedStreaming = Playlist(
```

Comment:
For consistency with other examples maybe `unifiedStreamingStreams` or was it intentional because of the _stream_ redundancy?